docs/usage/index.md

```diff
@@ -0,0 +1,91 @@
+![Renovate banner](https://app.renovatebot.com/images/whitesource_renovate_660_220.jpg){ loading=lazy }
```

Comment:
I'm ready for a draft review now. There may be other things we need to fix, like:

- [ ] autogenerating docs stuff
- [ ] stuff on the docs repository
- [ ] some config change
- [ ] some `nav` or `.pages` change
- [ ] docs test changes

The linters on this repository are happy now with the file, so at least we have that. :smile:

Reply:
Is there more I need to do, before I can mark this ready?